src/main/java/cinema/service/ShoppingCartServiceImpl.java

```diff
@@ -0,0 +1,46 @@
+package cinema.service;
+
+import cinema.dao.ShoppingCartDao;
+import cinema.dao.TicketDao;
+import cinema.lib.Inject;
+import cinema.lib.Service;
+import cinema.model.MovieSession;
+import cinema.model.ShoppingCart;
+import cinema.model.Ticket;
+import cinema.model.User;
+
+@Service
+public class ShoppingCartServiceImpl implements ShoppingCartService {
+    @Inject
+    private ShoppingCartDao shoppingCartDao;
+    @Inject
+    private TicketDao ticketDao;
```

Comment:
What for do you need this here?

Reply:
thought it will be used, forgot to remove it

Reply:
not what i meant but you got it right